magento/magento2
# [View] Removed extra space in link and script tag

### Description (*)
This PR helps to change extra spaces into single space in `<link>` and `<script>` tag.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
1. `./vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist lib/internal/Magento/Framework/View/Test/Unit/Page/Config/RendererTest.php`
2. Open the store and see `<link>` and `<script>` tag using view source code.
3. `<link>` and `<script>` should be without double spaces between attributes.

**Actual Result:**
![w_extraspace](https://user-images.githubusercontent.com/18184256/116742365-e13ef700-aa14-11eb-8c62-2df3548fa4fc.png)

**Excepted Result:**
![wo_extraspace](https://user-images.githubusercontent.com/18184256/116742657-485cab80-aa15-11eb-8396-005a36c80236.png)

### Questions or comments
Please let me your suggestions if any

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#32920: [View] Removed extra space in link and script tag
